deploy.gradle

```diff
@@ -191,12 +191,18 @@ task deploy {
 
                 def user = execute "whoami"
                 def host = execute "hostname -s"
+                def filesPath = System.getenv("FILES_DIR_PATH")
+                def javaPath = System.getenv("JAVA_DIR_PATH")
+                def temp = "$filesPath/$env/tmp"
```

Comment:
I would said that we should avoid calculated values and simple said that the tmp folder for beta is x, dev is y and prod is z.